### ISSUE TITLE
EXC_BAD_ACCESS KERN_INVALID_ADDRESS crash on 5.5.1

### ISSUE THREAD

**Issue by cojj90** [twilio/voice-quickstart-ios#419]
### Description

Hey guys,

We just got a crash report with this message:

```
EXC_BAD_ACCESS KERN_INVALID_ADDRESS 0x0000000000000008
resip::HandleManager::remove(unsigned long)
```

and the following stack trace:
```
Crashed: Thread
0  TwilioVoice                    0x1015cec28 resip::HandleManager::remove(unsigned long) + 2183104
1  TwilioVoice                    0x1015cf68c resip::Handled::~Handled() + 26 (Handled.cxx:26)
2  TwilioVoice                    0x1015d1024 resip::InviteSession::~InviteSession() + 2192316
3  TwilioVoice                    0x1015a2280 resip::ClientInviteSession::~ClientInviteSession() + 2000408
4  TwilioVoice                    0x1015ce180 resip::DumThread::thread() + 2180376
5  TwilioVoice                    0x1014ed314 threadIfThreadWrapper(void*) + 1259180
6  libsystem_pthread.dylib        0x1f2955b70 _pthread_start + 288
7  libsystem_pthread.dylib        0x1f295a880 thread_start + 8
```

#### Twilio Call SID(s)

CAa59caf883bac29850c19049c197ed4d7

### Versions

All relevant version information for the issue.

#### Voice iOS SDK

5.5.1

#### Xcode

11.3

#### iOS Version

14.1.0 (18A8395)

#### iOS Device

iPhone 6s Plus

**Comment by bobiechen-twilio** [comment]
Hi @cojj90 

Thanks for reporting the issue and I am sorry for the inconvenience.
Is it possible for you to share how often you see this crash in your app?

**Comment by bobiechen-twilio** [comment]
Hi @cojj90 

We have just released the [Voice iOS SDK 6.2.2](https://www.twilio.com/docs/voice/voip-sdk/ios/changelog#622), which includes a fix of this crash. Feel free to reach out to us if you need help upgrading to the latest SDK.

Best,
-bobie
